src/main/java/command/Command.java

```diff
@@ -0,0 +1,41 @@
+package command;
+
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Command is an abstract class that performs the command the user types into
+ * the chatbot.
+ */
+public abstract class Command {
+    private TaskList taskList;
```

Comment:
Perhaps you could consider using the access modifier `protected` instead so your child classes can access it without storing another copy in them?